save-cli/src/jvmTest/kotlin/org/cqfn/save/cli/GeneralTest.kt

```diff
@@ -27,9 +32,16 @@ import kotlinx.serialization.json.Json
 @OptIn(ExperimentalSerializationApi::class)
 class GeneralTest {
     private val fs = FileSystem.SYSTEM
+    private val json = Json {
```

Comment:
Can we reuse this `Json` from somewhere else? We could have `save-reporters` as a test dependency here (IIRC, there is the same polymorphic serializer there?)

Reply:
Refactored